doc/man3/PEM_read_bio_PrivateKey.pod

```diff
@@ -358,6 +358,10 @@ will be used.
 
 =head1 NOTES
 
+The PEM reading functions will skip any garbage or PEM data of a different
```

Comment:
I'm not sure _garbage_ is great here.  How about _extraneous context_ instead?

Reply:
fixup pushed (supposedly you've meant _extraneous content_)